docs/appendix/changelog.md

```diff
@@ -15,6 +15,41 @@ outputs:
 
 <!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->
 
+{{% changelog date="2024-03-01" color="added" title="Additional ML models" %}}
+
+Viam has added support for the TensorFlow, PyTorch, and ONNX ML model frameworks to the ML model service, expanding upon the existing support for TensorFlow Lite models.
+You can now [upload your own ML model](/ml/upload-model/) using any of these frameworks for use with the Vision service.
+
+{{% /changelog %}}
+
+{{% changelog date="2024-03-01" color="added" title="Payment and billing" %}}
```

Comment:
This wasn't added this month. I think this was all with GA but I can't say for sure